proto/felixbackend.pb.go

```diff
@@ -68,21 +68,16 @@
 */
 package proto
 
-import (
```

Comment:
Please run `make fix` then this diff will go away.  No sense in having everyone else get a diff after they run `make fix`

Reply:
done